assets/components/contribAmounts/contribAmounts.jsx

```diff
@@ -280,7 +282,7 @@ function getAttrs(props: PropTypes): ContribAttrs {
 
   if (props.contribType === 'ANNUAL') {
 
-    const userDefined = props.contribAmount.annual.userDefined;
+    const { userDefined } = props.contribAmount.annual;
```

Comment:
It really recommends we do this? This looks less clear to me...

Reply:
This is enforced by `eslint-config-airbnb`.